spec/helpers/curation_concern_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe CurationConcernHelper do
+  describe "#custom_h1" do
+    let(:gw) { GenericWork.new }
+    let(:bui) { BatchUploadItem.new }
+    let(:action_name) { "new" }
```

Comment:
you can probably take this out if you switch to hardcoding new into the helper method